src/components/LinkRenderer/LinkRenderer.module.css

```diff
@@ -0,0 +1,35 @@
+.linkWidthWrap {
+  position: relative;
+  max-width: 192px;
```

Comment:
Do we have some constant/var for this maybe?

Reply:
hm,  not really, it was specific value asked in the task for link width

Reply:
it would be good to have a prop "max-width" for LinkRendererShort (with 192 default value)
like how short link should be
at least it would be configurable